src/Microsoft.Diagnostics.Monitoring.RestServer/StorageOptions.cs

```diff
@@ -0,0 +1,20 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+#if UNITTEST
+namespace Microsoft.Diagnostics.Monitoring.UnitTests.Options
+#else
+namespace Microsoft.Diagnostics.Monitoring.RestServer
+#endif
+{
+    internal class StorageOptions
+    {
+        public string DumpTempFolder {get; set; }
```

Comment:
This will be really useful in partially alleviating #12 such that containers don't need to share their /tmp when using listening mode in dotnet-monitor, but they'll still need to mount an intermediate volume; at least now it can just share the dumps instead of everything in /tmp on every container.